### ISSUE TITLE
Add Shortcut for Copilot function

### ISSUE THREAD

**Issue by hkjang** [n4ze3m/page-assist#214]
Add Shortcut for Summarize and Explain and so on.

![image](https://github.com/user-attachments/assets/8cc0f13a-abf2-44e8-b552-2905841749ae)

**Comment by n4ze3m** [comment]
Copilot prompt for the selected items, btw

![image](https://github.com/user-attachments/assets/0ecff92d-8fed-475a-bbf6-8adf45816a8e)

**Comment by hkjang** [comment]
Yes, first of all, thanks for the guide. I already know that feature, but I think a shortcut would be better from a UX perspective since I have to move the mouse a lot.

**Comment by shishkin** [comment]
I'm also figuring out how to get a summary of a page with just one click or keyboard shortcut.
